composer.json

```diff
@@ -66,7 +71,8 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.0",
-        "guzzle/plugin-mock": "~3.1"
+        "phpunit/phpunit": "^9.0",
+        "guzzle/plugin-mock": "~3.1",
+        "symfony/phpunit-bridge": "^5.3"
```

Comment:
- PHPUnit upgrade
- Symfony PHPUnit bridge is a wonderful package that points out many code deprecations from Symfony packages